website/docs/usage/large-language-models.mdx

```diff
@@ -167,7 +173,113 @@ first time you use it. You can
 by setting the environmental variable `HF_HOME`. Also, you can upgrade the model
 to be `"databricks/dolly-v2-12b"` for better performance.
 
-### Example 3: Create the component directly in Python {id="example-3"}
+### Example 3: Add NER using a GPT-3 model from OpenAI {id="example-3"}
+
+In previous versions of the `spacy-llm` `NER` and `SpanCat` tasks, you could
+configure a zero-shot pipeline without any examples. The new `v3`
+(`spacy.NER.v3` and `spacy.SpanCat.v3`) tasks use few-shot learning exclusively
+require at least 1 prompt example to be configured.
+
+The old zero-shot prompt we were using really didn't work that well from our
+evaluations on well known NER datasets (# TODO: link to evaluation results?)
+
+The new v3 task prompts are based on the
+[PromptNER](https://arxiv.org/abs/2305.15444) paper and use chain-of-thought
+reasoning to improve model quality. The accuracy of this prompt should be much
+better for most NER cases but it does require a little bit more work upfront.
+
+Create a config file `config.cfg` containing at least the following (or see the
```

Comment:
Do we want to suggest having all of this in the config? Wouldn't it be better to read labels & label definitions from a file? It keeps the configs more generic.